js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadRoot.tsx

```diff
@@ -48,6 +48,44 @@ export const AssetLaunchpad = ({
   );
 };
 
+export const BackfillLaunchpad = ({
+  repoAddress,
+  sessionPresets,
+  assetJobName,
+  open,
+  setOpen,
+  onSaveConfig,
+}: {
+  repoAddress: RepoAddress;
+  sessionPresets?: Partial<IExecutionSession>;
+  assetJobName: string;
+  open: boolean;
+  setOpen: (open: boolean) => void;
+  onSaveConfig: (config: any) => void;
+}) => {
+  const title = 'Config Editor';
+
+  return (
+    <Dialog
+      style={{height: '90vh', width: '80%'}}
```

Comment:
These seem like reasonable bounds, though I think you'll want a `minWidth` (maybe 1000px or so) to make sure that a narrow viewport doesn't also show a weird, narrow editor.